pkg/apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -93,6 +90,14 @@ type CloneSetUpdateStrategy struct {
 	// Paused indicates that the CloneSet is paused.
 	// Default value is false
 	Paused bool `json:"paused,omitempty"`
+	// Priorities are the rules for calculating the priority of updating pods.
+	// Each pod to be updated, will pass through these terms and get a sum of weights.
+	PriorityStrategy *UpdatePriorityStrategy `json:"priorityStrategy,omitempty"`
+	// ScatterStrategy defines the scatter rules to make pods been scattered when update.
```

Comment:
We may slightly modify the comments saying that:
- Although priority strategy and scatter strategy can be applied together, we suggest to use either one of them.
- If scatterstrategy is used, we suggest to just use one term. 
Otherwise, the update order can be hard to understand.

Reply:
done